website/www/site/content/en/documentation/sdks/python.md

```diff
@@ -46,7 +46,13 @@ new I/O connectors. See the [Developing I/O connectors overview](/documentation/
 for information about developing new I/O connectors and links to
 language-specific implementation guidance.
 
-## Using Beam Python SDK in your ML pipelines
+## Making machine learning inferences with Python
+
+To integrate machine learning models into your pipelines for making inferences, use the RunInference API for PyTorch and Scikit-learn models. If you are using TensorFlow models, you can make use of the
```

Comment:
To integrate machine learning models into your pipelines for making inferences, use the RunInference API.....after this can we add a link to our repo just like how you did with the tfx_bsl here?

https://github.com/apache/beam/tree/master/sdks/python/apache_beam/ml/inference